File: introduction/ITP1_8_B-Sum of Numbers.java
import java.util.*;
public class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    String line;
    while(!(line = sc.nextLine()).equals("0")) {
      int sum = 0;
      for(int i = 0; i < line.length(); i++) sum += Character.getNumericValue(line.charAt(i));
      System.out.println(sum);
    }
  }
}

File: introduction/ITP1_8_B-Sum of Numbers.c
int main() {
  char line[1001];
  int sum, i;
  while(1) {
    scanf("%s", &line);
    if(line[0] == '0') break;
    sum = 0;
    for(i = 0; line[i]; i++) sum += line[i] - '0';
    printf("%d\n", sum);
  }
  return 0;
}
